src/cratedb/mod.rs

```diff
@@ -0,0 +1,130 @@
+use std::{borrow::Cow, collections::HashMap};
+
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "crate";
+const TAG: &str = "5.8";
```

Comment:
Thanks for supporting me on this. We should put the version the latest version [5.10.6](https://github.com/crate/docker-crate/commit/c03b3583d255cef7e4bf78e4159b6eb2e8d4fbe1)